codeforamerica/michigan-benefits
# Create views and flow for minimal SNAP application

**WHY**: This is an MVP. Next step will be to hook up filling out of PDF
and fax this.

To test:
* Visit root path
* Click "Apply now"
* Fill in all fields (all are required)
* Submit
* See success message on root path
* There should be a "MbSnapApp" record in the DB based on your entry
